onnx/test/shape_inference_test.py

```diff
@@ -1776,7 +1776,9 @@ def test_slice_with_input_shape_containing_dim_params(self) -> None:
                 make_tensor("ends", TensorProto.INT64, (3,), (1, 1, 1)),
             ],
         )
-        self._assert_inferred(graph, [make_tensor_value_info("y", TensorProto.FLOAT, (1, None, 1))])  # type: ignore
+        self._assert_inferred(
```

Comment:
it may be better to move this kind of formatting changes in a separate PR.

Reply:
I know. I wish I have a tool which can apply the same linting but lintrunner never worked for me.

Reply:
removed